meta-mbl-distro/recipes-extended/shadow/shadow-sysroot_%.bbappend

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+MBL_MIN_PASS_LEN ?= "5"
```

Comment:
Where does this `"5"` come from?

Reply:
It is the default:
https://github.com/shadow-maint/shadow/blob/4.6/etc/login.defs#L205

Reply:
Hmmm. I don't really like us repeating this in the recipe. We could just get rid of it and protect the `sed` command with:
```Shell
if [ -n "${MBL_MIN_PASS_LEN}" ]; then
```

I don't know whether it's worth it though... If you don't want to then can you add a comment explaining where the `"5"` comes from please?

Reply:
Addressed in 4a3571b.